images/macos/helpers/Common.Helpers.psm1

```diff
@@ -147,4 +147,17 @@ function Start-DownloadWithRetry
     }
 
     return $filePath
-}
\ No newline at end of file
+}
+
+function Add-EnvironmentVariable
+{
+    param
+    (
+        [Parameter(Mandatory)] [string] $Name,
+        [Parameter(Mandatory)] [string] $Value,
+        [string] $FilePath = "${env:HOME}/.bashrc"
+    )
+
+    $envVar = "export {0}={1}" -f $name, $value
+    Tee-Object -InputObject $envVar -FilePath $filePath -Append
```

Comment:
nitpick: $filePath -> $FilePath

Reply:
Do we need this output? Maybe replace to Add-Content.

Reply:
Fixed it